### ISSUE TITLE
installing bower dependencies hangs, never completes

### ISSUE THREAD

**Issue by onsombal** [Gottwik/enduro_mirror#4]
Hello, I'm attempting to install the enduro_mirror theme on macOS Sierra 10.12.5. The installer gets stuck on the step for "installing bower dependencies" as shown below.

 ┌—————————————————~—Enduro theme service—~
   —————————————————┐
   │ getting info for 'mirror' theme                           ✓ │
    ? choose project name: mirror
    ? choose your admin login: admin
    ? choose your admin password: [hidden]
   │ downloading and extracting theme                  ✓ │
   │ setting up admin credentials                            ✓ │
   │ getting project dependencies                           ✓ │
   │ installing npm dependencies                            ✓ │
   │ installing bower dependencies                          ⠙ │

Thoughts?

Thanks... Jim

**Comment by onsombal** [comment]
Turns out "sudo" needs to prefix the "enduro theme mirror" command.

**Comment by Gottwik** [comment]
Hey, good job on figuring it out.

Let us know if you find any other issues and good luck with the project 😄

**Comment by onsombal** [comment]
Thanks for the quick reply, Martin. Just getting into the details, but your
project is extremely impressive. My only suggestion is to dump Bower in a
future release.

I got all of the following projects running -- enduro_mirror, enduro_demo,
enduro_blog, and enduro_website. I ran into issues with legolabels because
I didn't bother installing mongo.db

BTW -- I also got the static pages for enduro_mirror up and running on both
Netlify and Github Pages.

My objective for enduro.js is to integrate it with my feathersjs + nuxt.js
project, which I'm also presently learning.

Regards... Jim
ᐧ
